packages/tasks/src/model-libraries.ts

```diff
@@ -661,6 +661,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: true,
 		countDownloads: `path:"pytorch_model.bin" OR path:"model.safetensors"`,
 	},
+	imstoucan: {
+		prettyLabel: IMS-Toucan",
+		repoName: "IMS-Toucan",
+		repoUrl: "https://github.com/DigitalPhonetics/IMS-Toucan",
+		countDownloads: `path_extension:"pt"`,
```

Comment:
For what I can see, https://huggingface.co/Flux9665/ToucanTTS/tree/main has a lot of files including two `.pt` files ([16kHz_encodec.pt](https://huggingface.co/Flux9665/ToucanTTS/blob/main/16kHz_encodec.pt) and [embedding_gan.pt](https://huggingface.co/Flux9665/ToucanTTS/blob/main/embedding_gan.pt)). This means that if the full repo is downloaded, we would be counting twice the number of downloads. In such a case it's preferable to set `path: "..."` with only one specific file. So the question you should wonder is "which files is guaranteed to be downloaded when a user instantiates the model?".

Reply:
The full repo is never downloaded, in the toolkit I only use hf_hub_download to download individual models from this bundle on demand. 

People might only want to use the pretrained vocoder, but train their own ToucanTTS checkpoint, or the other way around. That's why I chose the path_extension as filter. But you are right, it is too broad.

Is it ok to include more than one file? Then I would include the three most important files in the path. At least one of them will be downloaded by most users, but rarely more than one of them.

Reply:
That's perfectly valid and seems the most reasonable thing to do in your case yes. Here is an example on how to do it: https://github.com/huggingface/huggingface.js/blob/18231b70bd5435444ef3e1903b83a911dca1b374/packages/tasks/src/model-libraries.ts#L290

Reply:
Thank you! It is fixed now.